test/main_test.js

```diff
@@ -116,18 +118,23 @@ suite('End to End', () => {
   });
 
   test('test publish tarball', async function() {
+    //Setting fake credentials to bypass Taskcluster authentication.
+    let tempCreds = null;
+
     if (!credentials.clientId) {
-      this.skip();
+      tempCreds = {clientId: 'bypassTcCreds', accessToken: '123456'};
     }
+
     let doc = await documenter({
       project: 'docs-testing',
       schemas: validate.schemas,
       tier,
-      credentials,
+      credentials: tempCreds || credentials,
```

Comment:
I think you can just switch this out for `creds: {accessKeyId: 'fake', secretAccessKey: 'fake'}`